src/warp/multipass.rs

```diff
@@ -715,6 +746,53 @@ impl From<IdentityStatus> for warp::multipass::identity::IdentityStatus {
     }
 }
 
+#[derive(Clone, Copy)]
+#[wasm_bindgen]
+pub enum ImportLocationType {
+    /// Remote location where the identity is stored
+    Remote,
+
+    /// Local path where the identity is stored
+    Local,
```

Comment:
Might be worth nothing that wasm doesnt support filesystem so it might be best to have import and export from memory for `Local` instead. The code over in warp uses a little util that will be removed (since it shouldnt be exporting to localstorage for path in terms of a path)

Reply:
so only have `Remote` or am i misunderstanding? there is `Memory` but im not too sure how we should handle this on the web.

Reply:
`Memory` only stores it in a buffer that can then be used for whatever the caller wishes to use it for (in the case of uplink web, could trigger a download of the content being exported or provide a buffer that would be used to imported).